tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -292,12 +292,15 @@ private static async Task WriteDiagnosticLog(TracerManager instance)
                     writer.WritePropertyName("disabled_integrations");
                     writer.WriteStartArray();
 
-                    // For consistency with previous diagnostic log, this only includes integrations
-                    // that were disabled with DD_DISABLED_INTEGRATIONS, not integrations disabled with
-                    // DD_INTEGRATION_{0}_DISABLED
-                    foreach (var integration in instanceSettings.Integrations.DisabledIntegrations)
+                    // In contrast to 1.x, this only shows _known_ integrations, but
+                    // lists them whether they were disabled with DD_DISABLED_INTEGRATIONS,
+                    // or DD_INTEGRATION_{0}_DISABLED
```

Comment:
```suggestion
                    // or DD_TRACE_{0}_DISABLED
```

Reply:
Updated in https://github.com/DataDog/dd-trace-dotnet/pull/2048/commits/b464c316b326895ddb908f14421ca58a14d71a86